stack.c

```diff
@@ -53,16 +60,22 @@ int64_t stack_to_int(stack_value_t *entry, size_t size)
         value = entry->int_value;
         return value;
     }
+    /* int64_t */
+    case 8: {
+        int64_t value;
+        value = entry->long_value;
+        return value;
```

Comment:
Can you use `return (int64_t) entry->long_value` instead?

Reply:
ok, change in e4cc7c3